### PR TITLE
Rephrase "we" and "our"; structure options clearly.

### PR DESCRIPTION
**Pull request by osg-grafana**
Edit the _Getting started with Grafana Mimir_ guide.

### CODE REVIEW

File: docs/sources/operators-guide/getting-started/_index.md [hunk 1]
```diff
@@ -24,23 +27,24 @@ The written instructions focus on deploying Grafana Mimir as a [monolith]({{< re
...
 
-Download the appropriate [release asset](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
-For Linux with the AMD64 architecture:
+  Download the appropriate [release asset](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
 
-```bash
-curl -fLo mimir https://github.com/grafana/mimir/releases/latest/download/mimir-linux-amd64
-chmod +x mimir
-```
+  For Linux with the AMD64 architecture:
```
Comment (osg-grafana) [inline docs/sources/operators-guide/getting-started/_index.md:38]: Reviewers: This is a new line to make it more readable. No other content change.

File: docs/sources/operators-guide/getting-started/_index.md [hunk 2]
```diff
@@ -24,23 +27,24 @@ The written instructions focus on deploying Grafana Mimir as a [monolith]({{< re
 
 - Using Docker:
 
-```bash
-docker pull grafana/mimir:latest
-```
+  ```bash
```
Comment (osg-grafana) [inline docs/sources/operators-guide/getting-started/_index.md:30]: Indented to display code properly under the list item.

File: docs/sources/operators-guide/getting-started/_index.md [hunk 3]
```diff
@@ -7,13 +7,16 @@ weight: 10
 
 # Getting started with Grafana Mimir
 
-On this page, we provide two different options to getting started with Grafana Mimir. The written tutorial provides a series of imperative commands to start a single Mimir process. Our second tutorial, in video form, uses docker-compose to declaratively deploy multiple Mimir processes.
+There are two different options for getting started with Grafana Mimir:
+
+* The written tutorial provides a series of imperative commands to start a single Mimir process.
+* The visual tutorial (in the form of a video) uses Docker Compose to declaratively deploy multiple Mimir processes.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/getting-started/_index.md:13]:
```suggestion
* The visual tutorial (in the form of a video) uses `docker-compose` to declaratively deploy multiple Mimir processes.
```

**Comment by pstibrany** [comment]
I've rebased this PR on top of `main` to make all required `test` checks pass.

File: docs/sources/operators-guide/getting-started/_index.md [hunk 4]
```diff
@@ -7,13 +7,16 @@ weight: 10
 
 # Getting started with Grafana Mimir
 
-On this page, we provide two different options to getting started with Grafana Mimir. The written tutorial provides a series of imperative commands to start a single Mimir process. Our second tutorial, in video form, uses docker-compose to declaratively deploy multiple Mimir processes.
+There are two different options for getting started with Grafana Mimir:
+
+* The written tutorial provides a series of imperative commands to start a single Mimir process.
+* The visual tutorial (in the form of a video) uses `docker-compose` to declaratively deploy multiple Mimir processes.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/getting-started/_index.md:13]:
```suggestion
- The written tutorial provides a series of imperative commands to start a single Mimir process.
- The visual tutorial (in the form of a video) uses `docker-compose` to declaratively deploy multiple Mimir processes.
```

**Comment by osg-grafana** [comment]
> I've rebased this PR on top of `main` to make all required `test` checks pass.

Oops, just seeing this now.

**Comment by osg-grafana** [comment]
> > I've rebased this PR on top of `main` to make all required `test` checks pass.
> 
> Oops, just seeing this now.

I think I might have just broke this again because I did not know what you were working on the same thing.

**Review by chri2547 — APPROVED** [review]
LGTM!

**Pull request merged**